spec/decorator_spec.rb

```diff
@@ -0,0 +1,24 @@
+
+require_relative  '../decorator.rb'
+require_relative  '../capitalize_decorator.rb'
+require_relative  '../trimmer_decorator.rb'
+
+describe "Decorator design pattern Classes" do
+    $person = Person.new(1,18,"mohammedraad")
+    
+    context CapitalizeDecorator do
+    it 'capitalize the name' do
+      capitalize_decorator = CapitalizeDecorator.new($person)
+      name = capitalize_decorator.correct_name
+      expect(name).to eq("Mohammedraad")
+    end
+  end
+
+  context TrimmerDecorator do
+    it 'trim the name, limited to 10 char' do
+      trimmer_class = TrimmerDecorator.new($person)
+      name = trimmer_class.correct_name
+      expect(name).to eq("mohammedra")
+    end
+  end
+end
```

Comment:
When running the tests in this file, it is throwing this ⬇️ error. Kindly fix it and submit it for a re-review.
![image](https://user-images.githubusercontent.com/33268927/189051332-dee1828c-0b83-4bb1-bf5e-2840d99c2c2e.png)

Reply:
Hi @muneebulrehman 

Thanks for the review.
All the tests are passing in my terminal
![ScreenShot_20220908104830](https://user-images.githubusercontent.com/89518488/189066002-f7a3f9c8-0b50-496d-8dac-18e98887b971.png)

I will double-check the issue.

Best, 
Mohammed Raad,